src/ui/auth/pages/register/mnemonic-confirmation.tsx

```diff
@@ -25,46 +25,48 @@ const ListDiv = styled('div')({
 })
 
 const MnemonicConfirmation = ({ phrase, onConfirm }: MnemonicConfirmationProps) => {
-  const [selected, setSelected] = useState<string[]>([])
+  const [selected, setSelected] = useState<number[]>([])
   const words = useMemo<string[]>(() => shuffleArray(phrase.split(' ')), [phrase])
 
-  const isSelected = (word: string): boolean => {
-    return selected.indexOf(word) >= 0
+  const isSelected = (index: number): boolean => {
+    return selected.indexOf(index) >= 0
   }
 
-  const addSelected = (word: string) => {
-    setSelected([...selected, word])
+  const addSelected = (index: number) => {
+    setSelected([...selected, index])
   }
 
-  const removeSelected = (word: string) => {
+  const removeSelected = (index: number) => {
     const selectedCopy = [...selected]
-    selectedCopy.splice(selectedCopy.indexOf(word), 1)
+    selectedCopy.splice(selectedCopy.indexOf(index), 12)
```

Comment:
why deleteCount changed from 1 to 12?

Reply:
To be the same as in create account. There if the user clicks again on a selected word, then that word and all succeeding are deleted too (which maximum can be 12).

Reply:
ah, okay.